src/main/flight/pid.c

```diff
@@ -1007,7 +1007,7 @@ STATIC_UNIT_TESTED void rotateItermAndAxisError()
 {
     if (itermRotation
 #if defined(USE_ABSOLUTE_CONTROL)
-        || acGain > 0
+        || acGain > 0 || debugMode == DEBUG_AC_ERROR
```

Comment:
We normally wouldn't want a debugging mode to change flight behavior.  This is this even necessary? If the gain is 0 why do we need to log anything?

Reply:
With gain == 0 abs control doesn’t do anything. The reason I’m still doing the calculations if debug mode is ac_error is so we can see how much of an attitude error exists without abs control off. This should provide insight into how much of an improvement it brings under which circumstances.

LOS the benefits are very clear cut. I hope we can use this to work out when it will be most useful for FPV.